blog/2020-10-23-the-making-of-new-user-sdk-tutorials.md

```diff
@@ -0,0 +1,110 @@
+
+We now had a way to synchronize code with our documentation. But from where will the code be synchronized? We know users will likely want to view the source file and relative file path of the code snippet for added context. They will also be likely to clone the repo and try to run the sample.
+
+We actually already repositories of code samples for the [Go SDK](https://github.com/temporalio/samples-go) and [Java SDK](https://github.com/temporalio/samples-java). And while we desired to see more samples, there were already quite a few of them in each. But, we discovered that shared sample repositories tended to have two issues that made them less ideal for synchronizing with docs.
+
+1. While convenient, colocating multiple samples in a single repo is far less approachable compared with storing samples in self-contained repos.
+2. In shared sample repositories, it is harder to retain the idiomatic nature of any language and mirror the functionality of any sample across different languages.
+
+So for each of the sample applications we planned on using to drive our documentation, we created a corresponding template repo. These template repos can be easily copied, built, and run in a matter of minutes.
+
+- [Go money transfer template repo](https://github.com/temporalio/money-transfer-project-template-go)
+- [Go hello world template repo](https://github.com/temporalio/hello-world-project-template-go)
+- [Java money transfer template repo](https://github.com/temporalio/money-transfer-project-template-java)
+- [Java hello world template repo](https://github.com/temporalio/hello-world-project-template-java)
+
+## The tutorial
+
+<img alt="image" class="docs-image-centered docs-image-max-width-20" src="https://raw.githubusercontent.com/temporalio/documentation-images/main/static/tutorials.png" />
+
+Since the goal of our documentation changes was to help with new user acquisition, we decided to aim for a "tutorial" style of documentation. The first iterations aimed to build upon and replace the existing SDK “quick start” pages that maintained the status quo and printed “Hello World!” to the console. As you may have guessed, this route wasn’t sufficient enough to show users the real values that Temporal offers.
+
+Once it became clear that a standard approach wasn't going to cut it, we approached our co-founder and CEO, Maxim Fateev. We asked him to give us a demonstration that he typically uses to introduce engineers to Temporal for the first time. The scenario represents a money transfer from one bank account to another and he demonstrates what happens if one of the steps in the transfer fails. The money transfer sample was a great way to introduce the values of Temporal. For if you understand the ramifications of losing money from a failed financial transaction, several values of Temporal become immediately apparent:
```

Comment:
```suggestion
Once it became clear that a standard approach wasn't going to cut it, we brought in our co-founder and CEO, Maxim Fateev. We asked him to give us a demonstration that he typically uses to introduce engineers to Temporal for the first time. The scenario represents a money transfer from one bank account to another and during the demo Maxim showcases what happens if one of the steps in the transfer fails. The money transfer sample was a great way to introduce the values of Temporal. For if you understand the ramifications of losing money from a failed financial transaction, several values of Temporal become immediately apparent:
```

Reply:
So I think this sentence is indicative of a minimal but widespread issue. 

"For if you understand the ramifications of losing money from a failed financial transaction, several values of Temporal become immediately apparent:"

It's such a great message but its just wayyy too wordy. I feel like this is borderline shakespearian. I think this is true in quite a few other places. Anyway, just try and keep an eye on that.

Reply:
But this is also a Blog post - not docs.
I think that blog posts should not have to wait for the slowest reader - creative, though accurate, wording should be encouraged.